features/com.gwtplugins.eclipse.sdkbundle.gwt211.feature/feature.properties

```diff
@@ -0,0 +1,131 @@
+featureName  = GWT 2.11.0 SDK
+providerName = GWT Eclipse Plugin
+description = Downloads the GWT 2.11.0 SDK library 
+licenseURL = https://github.com/gwt-plugins/gwt-eclipse-plugin/blob/master/LICENSE.md
+
+# "license" property - text of the "Feature Update License"
+# should be plain text version of license agreement pointed to be "licenseURL"
+license=\
```

Comment:
Shouldnt the URL, license be of the contents of GWT? If the user is asking to download GWT, shouldn't we be letting them know the license of it?